src/Models/Organization.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Blumilk\Meetup\Core\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\Relations\HasMany;
+
+class Organization extends Model
+{
+    use HasFactory;
+
+    protected $fillable = [
+        "name",
+        "description",
+        "location",
+        "organization_type",
+        "foundation_date",
+        "number_of_employers",
+        "website_url",
+        "facebook_url",
+        "linkedin_url",
+        "instagram_url",
+        "youtube_url",
+        "twitter_url",
+        "github_url",
+    ];
+
+    protected $casts = [
+        "foundation_date" => "datetime:Y-m-d h:i:s",
```

Comment:
Maybe for that format `Y-m-d h:i:s` create class with const (`Formats` or smth like that).

Reply:
`"foundation_date:" . Formats::DATETIME,`